### DISCUSSION TITLE
Support for raw calldata instead of `args` and `funcName`

### DISCUSSION THREAD

**Discussion by xenide** [wevm/wagmi#1925] [Ideas]
Mainly in the context of `usePrepareSendTransaction` and `usePrepareContractWrite`. Cuz ethers has functions such as `Interface::encodeFunctionCall` which generates the calldata as a string and does not separate between funcName and args

**Comment by tmm** [comment]
Interesting. Why do you want to use this approach versus `functionName` and `args`?

**Reply by xenide** [reply]
it's mainly for multicall. Anyway the solution I figured out was to use `useSendTransaction`, which just takes in the raw calldata, and it works. 

Can resolve this if needed
